lib/src/drawing/elements/paths.dart

```diff
@@ -133,8 +133,8 @@ class MergePathsDrawable extends AnimationDrawable implements PathContent {
     }
 
     // TODO: figure out why this is broken.
-    // this is broken in android as well - just doesn't show up because it's usually disabled. this fixes some stuff for motorcycle.json
-    return Path.combine(PathOperation.union, firstPath, remainderPath);
+    // this is broken in android as well - just doesn't show up because it's usually disabled.
+    return Path.combine(op, firstPath, remainderPath);
```

Comment:
Does the `motorcycle.json` work with this from examples?

Reply:
let me take a look

Reply:
the problem seems to be limited to Intersect op, so I've added a temporary workaround for the intersect op (forcing it to .union) in order to display it at least as it was before. I can confirm though that Subtract is working as expected.

Reply:
The only thing I'm not sure of is where the bug is for that - is it a bug in motorcycle.json, or a bug in parsing, or a bug in Skia, or a bug in Flutter?  Either way, sounds good to me.  I disabled all of them because I didn't have good test cases.